src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -55,6 +55,8 @@ public sealed partial class PlatformCompatibilityAnalyzer : DiagnosticAnalyzer
         private const string IsPrefix = "Is";
         private const string OptionalSuffix = "VersionAtLeast";
         private const string Net = "net";
+        private const string macOS = nameof(macOS);
+        private const string MacSlashOSX = "MacOS/OSX";
```

Comment:
The `"m"` should be lower-case in `"macOS"`.

```suggestion
        private const string MacSlashOSX = "macOS/OSX";
```